### PR TITLE
Jwilaby/system activity via menu

### PR DESCRIPTION
**Pull request by justinwilaby**
This pr addresses parity with v3 regarding sending system activity events via menu items, #621

### CODE REVIEW

File: packages/sdk/ui-react/package.json [hunk 1]
```diff
@@ -26,8 +26,11 @@
     "tslint": "^5.10.0",
     "tslint-loader": "^3.6.0",
     "typescript": "2.7.2",
+<<<<<<< HEAD
     "webpack": "^4.10.2",
-    "webpack-cli": "2.1.3"
+=======
+>>>>>>> 7136538d2fb8e0f54cec8d638bd45e44bfd255f8
```
Comment (tonyanziano) [inline packages/sdk/ui-react/package.json]: Leftover merge conflict tags are breaking the build.

File: packages/app/main/src/data-v2/reducer/serviceUrl.spec.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { State } from '../state';
+import { Store } from 'redux';
+import { getStore } from '../store';
+
+describe('The serviceUrl reducer', () => {
+  let store: Store<State> = getStore();
+
+  test('should updated the store when a new serviceUrl action is dispatched', () => {
```
Comment (tonyanziano) [inline packages/app/main/src/data-v2/reducer/serviceUrl.spec.ts]: typo: `should update`*

Comment (justinwilaby) [reply packages/app/main/src/data-v2/reducer/serviceUrl.spec.ts]: +1

File: packages/sdk/ui-react/package.json (hunk 1 above)
Comment (justinwilaby) [reply packages/sdk/ui-react/package.json]: +1 (rebase woes)

**Pull request merged**